src/Lykke.AlgoStore.Job.Stopping/Settings/JobSettings/StoppingServiceCashSettings.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Lykke.AlgoStore.Security.InstanceAuth;
+
+namespace Lykke.AlgoStore.Job.Stopping.Settings.JobSettings
+{
+    public class StoppingServiceCashSettings
```

Comment:
I don't think we need to specify that the settings are cache related - there could be something else here